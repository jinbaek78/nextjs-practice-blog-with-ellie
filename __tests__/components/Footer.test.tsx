import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import Footer from '@/components/Footer';

describe.skip('Footer', () => {
  it('renders correctly', () => {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();
  });
});
