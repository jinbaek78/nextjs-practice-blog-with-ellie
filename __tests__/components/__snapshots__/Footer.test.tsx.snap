// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<div>
  <p
    class="bg-slate-900 text-white py-2 text-sm text-center"
  >
    Don't forget to CODE your DREAM | All Right Reserved.
  </p>
</div>
`;
